lib/screens/scholarship_screen/scholarship_detailsview.dart

```diff
@@ -0,0 +1,329 @@
+      }
+
+      if(current.scholarshipEligibility != null){
+        result.add(buildListingTile("Eligibility", current.scholarshipEligibility));
+      }
+
+    }on Exception catch(e){
+      print(e);
+      result = <Widget>[Center(child: Text("Something went wrong")),];
+    }
+    return result;
+  }
+
+  List<Widget> buildDetailContent(){
+    List<Widget> result = List.empty(growable: true);
+
+    try{
+
+      if(current.scholarshipCriteria != null){
+        result.addAll([buildListingTile("Criteria", current.scholarshipCriteria), buildDivider()]);
+      }
+
+      if(current.scholarshipMethod != null){
+        result.addAll([buildListingTile("Method Of Selection", current.scholarshipMethod), buildDivider()]);
+      }
+
+      if(current.scholarshipSpecial != null){
+        result.addAll([buildListingTile("Special Requirements", current.scholarshipSpecial), buildDivider()]);
+      }
+
+      if(current.scholarshipCondition != null){
+        result.addAll([buildListingTile("Condition", current.scholarshipCondition), buildDivider()]);
+      }
+      
+      if(current.scholarshipDetails != null){
+        result.add(buildListingTile("Additional Details", current.scholarshipDetails));
+      }
+      
+    }on Exception catch(e){
+      print(e);
+      result = <Widget>[Center(child: Text("Something went wrong")),];
+    }
+    return result;
+  }
+
+   void _showDialog(bool error) {
+      showDialog(
+        context: theme['context'],
+        builder: (BuildContext context) {
+          return !error ? AlertDialog(
+            titlePadding: const EdgeInsets.all(5),
+            contentPadding: const EdgeInsets.all(0),
+            title: ListTile(
+              title: Text("Copied to Clipboard"),
+              trailing: Icon(
+                Icons.check, 
+                color: Colors.green,
+              ),
+            ),
+          ): AlertDialog(
+            titlePadding: const EdgeInsets.all(5),
+            contentPadding: const EdgeInsets.all(0),
+            title: ListTile(
+              title: Text("Not Copied to Clipboard"),
+              trailing: Icon(
+                Icons.clear, 
+                color: Colors.red,
+              ),
+            ),
+          );
+        },
+      );
+    }
+
+  
+  @override
+  Widget build(BuildContext context) {
+    theme['context'] = context;
+    theme['theme'] = Theme.of(context);
+    theme['isDark'] = Provider.of<ThemeModel>(context,listen: false,).isDark;
+    return DefaultTabController(
+        length: 2,
+        child: Scaffold(
+          appBar: _buildAppBar(),
+          ///Note: Apparently there's a weird bug within the framework, when you select the last index
+          ///or tab and switch the orientation of the viewport/change dimensions, it creates an offset of the view, 
+          ///which can be reversed by scrolling the view. Hopefully, the fix comes soon =)
+          ///[https://github.com/flutter/flutter/issues/60288]
```

Comment:
Woah, that's unfortunate. Based on the last post the fix should definitely be on its way soon though.

Reply:
yeah =(